Playbooks/Zscaler-add-Domains-to-URL-Category/README.md

```diff
@@ -63,113 +50,163 @@ Hover over the **Activation** button and click **Activate** to enable the new Ro
- 
-Click the “Deploy to Azure” button and this will bring you to the Custom Deployment Template.
+You will need an integration account before this playbook can be deployed, as it is a requirement for executing JavaScript code, which is an operation used in the logic app.
 
-In the **BASICS** section:  
+Navigate to the Azure integration accounts page: https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.Logic%2FintegrationAccounts
 
-* Select the “**Subscription**” and “**Resource Group**” from the dropdown boxes you would like the playbook deployed to.  
+From the "**Overview**" page, select an existing integration account and take note of its name, or click "**Create**".
 
-In the **SETTINGS** section:   
+![Zscaler_Integration_Account_1](Images/Zscaler_Integration_Account_1.png)
 
-* **Integration Account Name**: This can be left as “Zscaler-Logicapp” or you may change it.  
+Select the subscription and resource group that this playbook will be deployed to and a name for the integration account, such as "**AS-Zscaler-Integration**". Review the region and select a pricing tier, then click "**Review + create**".
 
-* **Integration Account Pricing Tier**: You will need to select your Integration Account pricing tier here. Note that there are a limited number of free Integration Accounts you are allotted and you will need to select a higher tier if you have used these up.  
+![Zscaler_Integration_Account_2](Images/Zscaler_Integration_Account_2.png)
 
-* **Playbook Name**: This can be left as “Zscaler-add-Domains-to-URL-Category” or you may change it.  
+From the "**Review + create**" page, review the information, then click "**Create**".
 
-* **ZscalerURL**: Enter your Zscaler tenant URL here.
+![Zscaler_Integration_Account_3](Images/Zscaler_Integration_Account_3.png)
+
+From the deployment page, take note of the resource name of your integration account, as it will be needed for deployment.
 
-* **Zscaler Username**:  Replace text with username of the Zscaler Admin account you want to use. 
+![Zscaler_Integration_Account_4](Images/Zscaler_Integration_Account_4.png)
 
-* **Zscaler Password**: Replace text with password of the Zscaler Admin account you want to use. 
+#### Create Azure Key Vault Secrets:
 
-* **Zscaler API Key**: Replace text with API Key of your Zscaler org. 
+You will need to add you Zscaler API key and Zscaler password to an Azure key vault.
 
-* **Zscaler Custom URL Category Name**: Replace text with a Zscaler Custom URL Category Name. 
+Navigate to the Azure key vaults page: https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.KeyVault%2Fvaults
 
-Towards the bottom ensure you check the box accepting the terms and conditions and then click on “Purchase”. 
+Select an existing key vault or create a new one. From the key vault overview page, click the "**Secrets**" menu option, found under the "**Settings**" section. Click "**Generate/Import**".
 
```

Comment:
Same here, can you make these steps bulleted for readability?

Reply:
See previous comment